tests/test_models.py

```diff
@@ -1084,3 +1137,206 @@ def test_lots_of_sweeps(self):
             random_seed=2,
         )
         assert all(tree.num_roots == 1 for tree in ts.trees())
+
+
+class TestNeutralFixation:
+    """
+    Tests for the neutral fixation model.
+    """
+
+    def test_model_end_broken(self):
+        # Checking that we're correctly detecting the fact that
+        # sweeps are non renentrant.
+        model = msprime.NeutralFixation(
+            position=0.5, start_frequency=0.1, end_frequency=0.9, dt=0.01
+        )
+        with pytest.raises(RuntimeError, match="does not support interruption"):
+            msprime.sim_ancestry(10, model=model, end_time=0.0001)
+
+    def test_incorrect_num_labels(self):
+        model = msprime.NeutralFixation(
+            position=0.5, start_frequency=0.1, end_frequency=0.9, dt=0.01
+        )
+        for num_labels in [1, 3, 10]:
+            # Not the best error, but this shouldn't be exposed to the user anyway.
+            with pytest.raises(
+                _msprime.LibraryError, match="configuration is not supported"
+            ):
+                msprime.sim_ancestry(
+                    10,
+                    model=model,
+                    num_labels=num_labels,
+                )
+
+    def test_float_position_discrete_genome(self):
+        # We can still specify a floating point position if we want, doesn't
+        # make any difference.
+        model = msprime.NeutralFixation(
+            position=0.5, start_frequency=0.1, end_frequency=0.9, dt=0.01
+        )
+        ts = msprime.sim_ancestry(
+            10, sequence_length=1, recombination_rate=20, model=model
+        )
+        assert ts.num_trees == 1
+
+    def test_neutral_fixation_no_recomb(self):
```

Comment:
is this test helpful?  If it failed, that wouldn't imply an error in the neutral fixations, but somewhere else in the library, right?

Reply:
fair. can probably be removed.